src/Web/src/components/layout/Layout.tsx

```diff
@@ -1,51 +1,55 @@
-import React, { Component } from "react";
+import React, { useEffect } from "react";
 import NavBar from "../navBar/NavBar";
 import Footer from "../footer/Footer";
 import LayoutRouter from "./components/LayoutRouter";
-import { Redirect, RouteComponentProps, withRouter } from "react-router";
+import { RouteComponentProps, withRouter } from "react-router";
 import { ToastContainer } from "react-toastify";
 import { connect } from "react-redux";
 import { DispatchProps, mapDispatchToProps, mapStateToProps, StateProps } from "./store/PropsTypes";
 import LoaderSplash from "../ui/loaderSplash/LoaderSplash";
+import { CssClass } from "./enums/CssClass";
```

Comment:
Podobnie, skoro to jest tylko jedna klasa do main wydaje mi się że troche niepotrzebne komplikowanie

Reply:
Z jednej strony zaczęło mi się bardziej podobać rozbijanie na większa ilość plików, ale masz rację - jedna konwencja na caly projekt.

Reply:
Druga kwestia jest taka - enumy zostają w kompilacji zmienione na zwykły string, natomiast tak jak robimy teraz, zawsze jest to kolejna zmienna trzymana w pamięci